devtools/server/actors/replay/connection.js

```diff
@@ -1082,6 +1111,137 @@ Services.ppmm.addMessageListener("RecordingStarting", {
   },
 });
 
+function getChannelRecording(channel) {
```

Comment:
It would be nice to have an overview comment here describing why we need to capture network data in the parent process and then send messages to the recording process describing what happened.